src/command/client/search/event.rs

```diff
@@ -1,48 +1,46 @@
 use std::{thread, time::Duration};
 
 use crossbeam_channel::unbounded;
-use termion::{event::Event as TermEvent, event::Key, input::TermRead};
+use crossterm::event::{poll, read, Event};
 
-pub enum Event<I> {
+pub enum AtuinEvent<I> {
```

Comment:
Renamed this to avoid confusion in `interactive.rs`.

Maybe `EventWrapper` would be better.